packages/xs-extra/xapi-plugin.master/opam

```diff
@@ -9,6 +9,8 @@ remove: [
   ["ocamlfind" "remove" "worker"]
 ]
 depends: [
-  "obuild" {build}
+  "oasis" {build}
+  "ocamlbuild" {build}
+  "base-unix"
   "rpc"
```

Comment:
I did not do it because we only use the legacy rpclib functions in there, no new ppx-related stuff, so it makes no difference in the end